Makefile

```diff
@@ -186,4 +210,17 @@ navigation-fixtures:
 
 .PHONY: 1.0-ui-publish-to-artifactory
 1.0-ui-publish-to-artifactory:
-	./gradlew :libnavigation-ui:artifactoryPublish
\ No newline at end of file
+	./gradlew :libnavigation-ui:artifactoryPublish
+
+.PHONY: 1.0-ui-check-api
+1.0-ui-check-api:
+	# TODO Remove -PhideId=ReferencesHidden after fixing errors
+	./gradlew :libnavigation-ui:checkApi -PhidePackage=com.mapbox.navigation.ui.internal -PhideId=ReferencesHidden
+
+.PHONY: 1.0-ui-update-api
+1.0-ui-update-api:
```

Comment:
nit: think we could have 2 commands that cover public apis

make check-api
make update-api

Reply:
talking about it offline some too.

1.0-ui + 1.0-core is consistent with the other commands